5_databases/week2/homework

```diff
@@ -0,0 +1,145 @@
+INSERT INTO task (title, description, created, updated, due_date, status_id)
+VALUES ('Learn an  instrument', 'Maybe violin', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP, '2023-12-31', 1);
+
+
+UPDATE task
```

Comment:
Correct, but better to do with a primary key (id) so you dont edit more than one by accident